src/Resources/TortoiseProcENG.rc

```diff
@@ -339,7 +339,7 @@ BEGIN
     GROUPBOX        "TortoiseGit",IDC_STATIC,7,7,286,92
     LTEXT           "&Language:",IDC_STATIC,14,20,86,8
     COMBOBOX        IDC_LANGUAGECOMBO,108,18,176,64,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    CHECKBOX        "Automatically check for &newer versions every week",IDC_CHECKNEWERVERSION,14,34,195,26,BS_VCENTER | BS_MULTILINE
+    CHECKBOX        "Automatically check for &newer versions every week",IDC_CHECKNEWERVERSION,14,36,195,18,BS_VCENTER

```

Comment:
The reason I added this change is the checkbox's height is too tall and the control background overlaps to `Re-run First Start Wizard` button.

Please check below screenshot images.

![ws000000](https://cloud.githubusercontent.com/assets/1131125/21284473/6bb56df6-c45f-11e6-88d2-b8a5c5be6af3.png)
![ws000001](https://cloud.githubusercontent.com/assets/1131125/21284474/6bb9b42e-c45f-11e6-970c-324a449781f4.png)